homework_4.py

```diff
@@ -0,0 +1,57 @@
+test_info_1 = "Please, write your name: "
+test_info_2 = "Write the letters you want to delete from your name?: "
+test_info_3 = "Your new name is: "
+answer_info_1 = input(f"{test_info_1}")
+answer_info_2 = input(f"{test_info_2}")
```

Comment:
This is an invalid way for the input() function
You can write:
answer_info_1 = input("Please, write your name: ")

Reply:
fixed